docs/source/upgrade/example/coinV2/daml.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+sdk-version: 0.0.0
+# BEGIN
+name: coinV2
+parties: [Alice, Bob]
+dependencies:
+  - daml-prim
+  - daml-stdlib
+  - ../coinV1/.daml/dist/coinV1-0.0.1.dar
```

Comment:
Let’s please use `data-dependencies` and `--target=1.8` so this works cross-SDK.

Reply:
ok.
Does the explicit `--target=1.8` get stripped out when me bump to that version?

Reply:
Yeah once we switch the default to `1.x`, I usually grep for all `--target=1.x` flags and strip them out unless there is a reason for locking things explicitly to that version.

Reply:
I tried `--target-1.8` but it didn't work. So for now I used `--target=1.dev`.

Reply:
How did it fail? Are you sure you rebased on master and used a compiled built from HEAD? If it actually doesn’t work something is very wrong in `1.8` so let’s please investigate and not simply change to `1.dev`.

Reply:
I did just test building this on `1.8` and it worked fine so my suspicion is that you might have tried this before rebasing or using the latest release instead of `daml-sdk-head`.

Reply:
Doh! -- rebased on master; but forgot to use `daml-head`.
Now see 1.8 is good.
Sorry for the noise.